roles/openshift_version/tasks/check_available_rpms.yml

```diff
@@ -1,7 +1,7 @@
 ---
-- name: Get available {{ openshift_service_type}} version
+- name: Get available RPM version
   repoquery:
-    name: "{{ openshift_service_type}}{{ '-' ~ openshift_release ~ '*' if openshift_release is defined else '' }}"
+    name: "{{ openshift_service_type }}{{ '-' ~ openshift_version ~ '*' if openshift_version is defined else '-' ~ openshift_release | default('') ~ '*' }}"
```

Comment:
Is 'atomic-openshift-*' valid for this case.  I think it probably is, but previously we were just looking for 'atomic-openshift' if openshift_release was undefined.